scripts/CubitExport-release.cmake

```diff
@@ -0,0 +1,190 @@
+#----------------------------------------------------------------
```

Comment:
Do we need this file? or is it part of the SDK?

Reply:
yes it is part of the sdk, I had to tweak it to have it work on Mac, 
some paths were not provided correctly...

Reply:
I wonder if we should provide that differently then???  It seems to make more sense to patch the SDK that include the whole file here, but I'm not sure the best way to do that....

Reply:
we could get a diff, and apply it ?

Reply:
That sounds like a good plan to me @bam241!